webapp/app/css/treeherder.css

```diff
@@ -418,10 +418,30 @@ th-watched-repo {
     display: flex;
     display: -webkit-flex;
     -webkit-flex-flow: row wrap;
+    flex-flow: row nowrap;
+    align-items: center;
+    justify-content: space-between;
+}
+
+.result-set-left {
+    display: flex;
+    display: -webkit-flex;
+    -webkit-flex-flow: row wrap;
     flex-flow: row wrap;
+    flex: auto;
     align-items: center;
 }
 
+.result-set-progress {
+    color: #6f6d70;
+    font-style: italic;
+    margin-left: 10px;
```

Comment:
I just tried the branch (super nice!) but ya, I agree here. Perhaps it could float at the mid-point between the revision and the right hand UI elements, or something.